stacks/SiriGeneratorStack.ts

```diff
@@ -11,10 +11,14 @@ export function SiriGeneratorStack({ stack }: StackContext) {
         name: `cdd-siri-sx-${stack.stage}`,
     });
 
+    const siriSXUnvalidatorBucket = new Bucket(stack, "SiriSXUnvalidatedBucket", {
+        name: `cdd-siri-sx-unvalidated-${stack.stage}`,
+    });
+
     const siriGenerator = new Function(stack, "cdd-siri-generator", {
         environment: {
-            SIRI_SX_BUCKET_ARN: siriSXBucket.bucketArn,
-            SIRI_SX_BUCKET_NAME: siriSXBucket.bucketName,
+            SIRI_SX_UNVALIDATED_BUCKET_ARN: siriSXUnvalidatorBucket.bucketArn,
```

Comment:
It doesn't look like we're using the ARN so this can be removed

Reply:
done and removed where required